RELEASE.md

```diff
@@ -0,0 +1,101 @@
+# Release process
+
+This document is for maintainers to release versions of the GHPRB plugin.
+
+The following release process is a little intense.  However, it serves as a high
+quality assurance test which users will appreciate when they go to upgrade their
+own Jenkins and nothing breaks from GHPRB.
+
+- [Before releasing](#before-releasing)
+- [Steps to release](#steps-to-release)
+
+# Before releasing
+
+Summary:
+
+1. Setup Jenkins with GHPRB.
+2. Track `$JENKINS_HOME` configuration with git.
+3. Upgrade the plugin and check for any differences in `$JENKINS_HOME` which
+   would negatively affect users. The configs should not break and Jenkins
+   should cleanly migrate existing configurations.  That is, configured settings
+   and jobs should be able to upgrade.
+
+### 1. Setup Jenkins with GHPRB
+
+Test that the plugin properly upgrades configuration without breaking existing
+configuration.  Provision Jenkins LTS by downloading `jenkins.war`.  Start
```

Comment:
BTW you can automate this kind of thing with `@Rule JenkinsRule` + `@LocalData`.

Reply:
I'm not familiar with those annotations.  Is there a wiki document where I can learn more?